### DISCUSSION TITLE
Part 3. Design: Perceptions of design

### DISCUSSION THREAD

**Discussion by Erioldoesdesign** [sprblm/USER_project#73] [Findings discussions]
### Introduction
Perceptions of design in SROSS vary widely across people and projects, as per their lived experiences, governed by team dynamics, decision making processes, the role design plays in their work, among other factors. Design is subjective, hence, there is no right or wrong with how it is understood. When asked what design meant to them, our research participants came up with a broad set of responses ranging from design being an activity that is centered around users and community, enables collaboration, brings clarity, is visual, promotes thoughtfulness, yet is sometimes ignored, treated as optional and is aspirational.
Design is thinking about users and community
Design enables users to find their way to get started or ‘onboarded’ onto the software then nudges them towards the right workflows. Design focuses on human experience, makes the software inclusive for all users, reaching even those without technical proficiency, ones who might not want to dive into the code - since writing code is a prominent method of interacting with a science & research open source software. Design process involves the broad stakeholder community from start to finish and considers different perspectives.

### Design is all about communication by challenging assumptions and bringing clarity
One of the participants who identified as a programme director thinks “that's where design is super challenging to researchers, because they already start with a solution to the problem they want to solve. Stepping back to the basics of design, means they have to completely undo all their assumptions in their structures.” Assumptions may not only be present in the way the software is structured but also the language and terminology, which users may not be aware of. Assumptions may lead to disparity between how the software is programmed and how the users actually use it, which then is the design’s role to bridge by “[understanding] the nature in while people interact with different features of the software” and then offering choices in a visible, actionable and clear manner. Design is thus all about communicating by unpacking assumptions, using a common language, representing the true state of the application and presenting information that’s easily accessible.

### The visual appeal, aesthetics, attractiveness, prettiness and branding of the software are all seen as design
Design is often seen in terms of ‘look and feel’ of the application, and common terms used in this context are ‘fonts’ and ‘colors’. While there are other perceptions of designs mentioned above, those who see design in the box of visuals and aesthetics, often do not bring up the other values of design. One respondent expressed that “design was almost put into the box of like, design is branding. Design is decorative, it has zero relevance to our, you know, hardcore dev community. So we'll just put this in the box and like, leave it out there.” On the contrary, another respondent observed that the visual aspect of design is rooted in psychology, and operates from the psychology of interfacing with the users which implies that it serves beyond the cosmetic value. A further participant that performs a design function cross multiple SROSS project adds to these thoughtful and exploratory perceptions of design when describing the changes that design has had by saying that the perception of design have changed and evolved as compared to the past into more than being “just colors, fonts, what kind of images we have in the layout”.

### Design is also seen as the planning part of developing the software, a preparatory step in understanding the need and structure of the software
This take on design has a technical angle to it. One of the participants who identified as a researcher and maintainer explained their idea of design as “picturing how to structure the configuration files for our software, and also the outputs and what plots people are going to want to see. Trying to think about how people will use my software, how will I and others build on my software in the future.” Another participant who is a master’s student mentioned that design is “understanding what you need, before even delving into building the software”. The structure, components and architecture of the software involve more technical and programming skills than interface conceptualisation skills. This perspective goes on to show the broad range of definitions that design takes.

### Design helps empowers and helps fulfill the purpose of research
All functions of software development should help the outcomes and the goal for which it’s built. The ability for design to empower research was heard from only a couple of participants. "Enabling researchers to use technology to handle, discover, browse, and present their data to the general public through design” as one UX/UI designer participant mentioned, helps fulfill the purpose of research. An element of thoughtfulness and design does not just go into the process of research but also the research output, so that questions such as “how is research applied? How do people engage with it? How do you publish it? How do you make it open?” can be answered.

### It is hard to engage with design experts because of the prevailing structures in open sciences and lack of frameworks, even when there is interest. 
There is a perception that designers and scientists live in different spheres and that their metrics of success in SROSS are extremely different. Seeing something as ‘other’ or ‘different’ resists collaboration. As mentioned earlier, this perception piggybacks on design’s ability to only deliver cosmetic value through animations, billboards, advertisements and the like. This wall between science and design is built on the foundations of grant styles which is palpable in the remark of one of the researcher participants who said that they had “never seen an academic grant include design, in fact people will look at you funny if you try.” In a similar vein another participant explained to us how “academic grants typically give [projects] money to hire postdocs and PhD students. So you cannot easily embed someone that comes from a design perspective into your lab as there's no framework for that.” There have been innovative attempts to include design by pitching it as the “co-creation of project with the community” and there’s significant desire to improve user experience but “how to make that happen is a bigger and harder question.”

### Conclusion
There’s truly a wide range of how design is perceived. It is often considered synonymous to accessibility. It is perceived as making users’ lives easier. It serves by optimizing and making software beautiful. Sometimes it can even be challenging to put specific words to explain design. Being aware of the different ways people understand design can help in building a more consistent and shared language among the community. Awareness can also promote a more calculated decision on making design a part of OS development.

**Our recommendation/s:**
**Building a shared understanding of design:** How it adds value to the project and what is expected from a designer will support research and enable the fulfillment of research objectives.
**Artifacts can change minds:** Stakeholders can be quickly convinced, if artifacts are produced that enable SROSS’s purpose and work and not distract them from it. For example, prioritizing the design of relevant evidence-led features and critical users usability as opposed to perpetuating an untrue stereotype that designers are most interested in ‘pixel perfection’ and ‘button radius’ can enrich the collaboration and help the mutual respect between SROSS and design.
**Design in SROSS can be a lonely activity:** Supporting team members who look after the design aspects will eventually produce better value.

 2. This quote is also highlighted in design work in the academic context.

**Comment by Mohautheko97** [comment]
Inspiring our future leaders in following their ambitiouns, finding their paths and being able to focus on the things that make them unique. Some are good with their hands(handcraft/statue-designer, etc ) minded (visualiser/seeir/minecraft) creators (drafting/color-cording/setting /skimming and inventory) this is the kind of people who make a brand be  incredible and if their combination brings beautiful-delicately and desirable design that shakes the entire feild of branding in the business industry that the reason why we are the best.

**Reply by Erioldoesdesign** [reply]
A beautiful summary statement of creatives :) thanks
